src/coreclr/gc/windows/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+include_directories("../env")
+include_directories("..")
+
+set(GC_PAL_SOURCES
+    gcenv.windows.cpp
+)
+
+add_library(gc_pal STATIC ${GC_PAL_SOURCES} Native.rc)
```

```suggestion
add_library(gc_pal STATIC ${GC_PAL_SOURCES})
```
Version resources should not be needed for static libraries.